rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -622,6 +624,27 @@ def test_find_yearn_vaults_v2_price(inquirer_defi, globaldb):
             assert result and result[0].address == underlying_token.resolve_to_evm_token().evm_address  # noqa: E501
 
 
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_find_gearbox_lp_price(inquirer: 'Inquirer', arbitrum_one_manager: 'ArbitrumOneManager'):
+    dwethv3 = EvmToken('eip155:42161/erc20:0x04419d3509f13054f60d253E0c79491d9E683399')
+    sdwethv3 = EvmToken('eip155:42161/erc20:0x6773fF780Dd38175247795545Ee37adD6ab6139a')
```

Comment:
shouldn't this test be vcred since it does external calls? Also since you will have it vcred assert for the right price